api/v1beta1/hostedcluster_types.go

```diff
@@ -446,6 +449,26 @@ const (
 	GuestOLMCatalogPlacement OLMCatalogPlacement = "guest"
 )
 
+func (olm *OLMCatalogPlacement) String() string {
+	return string(*olm)
```

Comment:
can you link which interface is this implementing?

Reply:
https://github.com/spf13/pflag/blob/d5e0c0615acee7028e1e2740a11102313be88de1/flag.go#L185-L191

Reply:
Yep, that's correct